vumi/transports/dmark/dmark_ussd.py

```diff
@@ -113,8 +113,20 @@ def handle_raw_inbound_message(self, request_id, request):
             log.msg('Unhappy incoming message: %r' % (errors,))
             self.finish_request(
                 request_id, json.dumps(errors), code=http.BAD_REQUEST)
+            yield self.add_status(
+                component='request',
```

Comment:
I think we decided on making it separate, in case a request comes in that's okay (we send an ok status), then if we timeout sending a response (we send a down status), and it would toggle between sending those two statuses for every message.